Dockerfile

```diff
@@ -37,7 +37,7 @@ ENV DEVISE_SECRET boopboop
 ENV SECRET_COOKIE_TOKEN beepbeep
 
 # build assets
-RUN bundle exec rake deploy:build
+RUN bundle exec rake deploy:build docker
```

Comment:
This is definitely the right approach, but I found I needed 

```
RUN bundle exec rake deploy:build[docker]
```

to make this go. Does passing `docker` in like that work for you?